src/modules/i18n/locales/de.json

```diff
@@ -658,5 +658,7 @@
     "operation_successful": "Erfolg!",
     "operation_failed": "Hoppla! Etwas ist schiefgegangen. Bitte versuchen Sie es erneut.",
     "operation_aborted": "Operation abgebrochen",
-    "outside_valid_year_range": "Das eingegebene Jahr liegt außerhalb des verfügbaren Bereichs:"
+    "outside_valid_year_range": "Das eingegebene Jahr liegt außerhalb des verfügbaren Bereichs:",
```

Comment:
In CH-german, please use a double `s` instead of `ß` in `außerhalb`

Reply:
I've changed this in the translation table, but will update the code in a separate PR as this translation is not part of this PR and that other open PR have changed translation.